conformance/Makefile

```diff
@@ -30,8 +31,10 @@ preload-nginx-container: ## Preload NGINX container on configured kind cluster
 
 .PHONY: build-and-load-images
 build-and-load-images: preload-nginx-container ## Build NKG container and load it and NGINX container on configured kind cluster
+ifeq ($(BUILD_NKG),true)
 	yq -i 'with(.spec.template.spec.containers[0]; .image = "$(NKG_PREFIX):$(NKG_TAG)" | .imagePullPolicy = "Never")' $(NKG_DEPLOYMENT_MANIFEST)
```

Comment:
Maybe this step belongs outside of this make target (relating to my question above).

Reply:
Updated the make targets